README.md

```diff
@@ -22,6 +22,12 @@ To generate the documentation
 lein doc
 ```
 
+## Docs
+
+- [Docs](doc/)
+- Tutorials:
+  - [Modeling Threat Intelligence in CTIM](doc/tutorials/modeling-threat-intel-ctim.md)
```

Comment:
I think that only the main doc directory should be referenced in the main `README.md`. A tutorial section could be added to the `doc/README.md` file